keyboards/contender/readme.md

```diff
@@ -0,0 +1,19 @@
+# contender
+
+![](https://raw.githubusercontent.com/sotoba/contender/images/contender_topview.jpg)
+
+A number pad for hardboot.
+
+* Keyboard Maintainer: [sotoba](https://github.com/sotoba)
+* Hardware Supported: Contender PCB
+* Hardware Availability: [BOOTH: stupa-devices](https://stupa-devices.booth.pm/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Add Bootloader section.

https://github.com/qmk/qmk_firmware/pull/15403/files#diff-2b191cee3aa81d3a83a62fe182bf3938ab7cad04d7c2d3c654b1c8d9a60fd853R21